packages/cli/src/handlers/login.ts

```diff
@@ -90,5 +91,13 @@ export const login = async (url: string, options: LoginOptions) => {
         : await loginWithPassword(url);
     await setContext({ serverUrl: url, apiKey: token });
     await setDefaultUser(userUuid);
-    await setProjectInteractively();
+    try {
+        await setProjectInteractively();
+    } catch (e) {
```

Comment:
Should we have a condition to show the error message if the error code isn't 404 ?